index.html

```diff
@@ -1635,9 +2193,625 @@ <h2 id="AudioContext">
             output latency. Lowest power consumption.
           </dd>
         </dl>
+-->
+        <pre class="idl">
+[Constructor,
+ Constructor(optional AudioContextOptions contextOptions)]
+interface AudioContext : BaseAudioContext {
+    readonly        attribute double baseLatency;
+    readonly        attribute double outputLatency;
+    AudioTimestamp                  getOutputTimestamp ();
+    Promise&lt;void&gt;                   suspend ();
+    Promise&lt;void&gt;                   close ();
+    MediaElementAudioSourceNode     createMediaElementSource (HTMLMediaElement mediaElement);
+    MediaStreamAudioSourceNode      createMediaStreamSource (MediaStream mediaStream);
+    MediaStreamTrackAudioSourceNode createMediaStreamTrackSource (MediaStreamTrack mediaStreamTrack);
+    MediaStreamAudioDestinationNode createMediaStreamDestination ();
+};              
+        </pre>
+        <section>
+          <h2>
+            Constructors
+          </h2>
+          <dl class="methods" data-dfn-for="AudioContext" data-link-for=
```

Comment:
@marcoscaceres What's the proper way to define constructors? I just used class="methods", but I think that might be wrong. Maybe data-dfn-for and data-link-for are also wrong.

Reply:
Unfortunately, I've not linked up constructors properly yet in ReSpec. This is what we do in `PaymentRequest` API. If you do the same, it will auto link correctly once I fix that:

```HTML
        <p>
          The <code><dfn data-lt=
          "PaymentRequest.PaymentRequest()">PaymentRequest(<var>methodData</var>,
          <var>details</var>, <var>options</var>)</dfn></code> constructor MUST
          act as follows:
        </p>
```

The key part is that `PaymentRequest.PaymentRequest()` is the actual constructor, because it's just a function.

Reply:
Yeah, I knew constructors weren't linked.  And from the PaymentRequest, I see that constructors aren't defined using any special `<dl class="foo">` construct, so perhaps I should just rip this out.

You're not going to add something like `<dl class="constructor">` or something to match the attribute and methods dl classes?